geometry_msgs/PoseStamped place_position
---
bool result
---
std_msgs/Float32 linear_error
std_msgs/Float32 linear_velocity
std_msgs/Float32 angular_error
std_msgs/Float32 angular_velocity